sharepoint/sharepoint-ps/sharepoint-online/Start-SPODataAccessGovernanceInsight.md

```diff
@@ -255,6 +268,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -UserIDList
+
+Specifies the AAD/Entra object IDs of the users for whom permissions report should be generated. Can be fetched using the ```Get-MgUser``` command from Microsoft Graph PowerShell.
```

Comment:
>Microsoft Graph PowerShel [](http://example.com/codeflow?start=152&length=25)

you could href link this:

[Microsoft Graph PowerShell](/powershell/module/microsoft.graph.users/get-mguser)